dj_beneficiary/models.py

```diff
@@ -315,19 +298,6 @@ class IndividualBeneficiary(models.Model):
         null=True,
         blank=True,
     )
-    agent = models.ForeignKey(
-        Agent,
-        on_delete=models.PROTECT,
-        null=True,
-        blank=True
-    )
-    registered_facility = models.ForeignKey(
-        'Facility',
-        on_delete=models.PROTECT,
-        null=True,
-        blank=True,
-        related_name="registerd_facility"
-    )
```

Comment:
Lets leave facility model as it is  without abstructing it
that way we can reference to it in these abstract model definitions

Reply:
My initial thought was that the relationships should be handled where the abstract models are being used, but I will revert it.